extensions/amp-social-share/1.0/social-share.js

```diff
@@ -70,48 +83,58 @@ export function SocialShare(props) {
  *   baseEndpoint: ?string,
  *   checkedWidth: ?number,
  *   checkedHeight: ?number,
+ *   params: !JsonObject,
  * }}
  */
 function checkProps(props) {
   const {
     'type': type,
-    'shareEndpoint': shareEndpoint,
-    'params': params,
-    'bindings': bindings,
+    'endpoint': endpoint,
+    'target': target,
     'width': width,
     'height': height,
+    // eslint-disable-next-line no-unused-vars
```

Comment:
Lots of lint disables here. IMHO it'd be better to just use omit. But before we do that, see my comments on `finalEndpoint = ...`

Reply:
Sounds good, thanks will update!